messages/app/models/templates.py

```diff
@@ -0,0 +1,21 @@
+from . import db
+import uuid
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class Template(db.Model):
+    __tablename__ = "templates"
+
+    UUID = db.Column(
+        str(UUID(as_uuid=True)),
+        db.String,
```

Comment:
Вроде бы можно просто:
```suggestion
        UUID(as_uuid=True),
```